android/src/main/java/com/onegini/mobile/sdk/reactnative/handlers/InitializationHandler.java

```diff
@@ -1,7 +1,7 @@
 package com.onegini.mobile.sdk.reactnative.handlers;
 
 import com.onegini.mobile.sdk.android.handlers.error.OneginiInitializationError;
-
+//Does it has to be java file?
```

Comment:
This file doesn't even need to exist as far as i can see

Reply:
Then burn it with fire 🔥